### PR TITLE
fix ignore emergency parameter not working

### PR DESCRIPTION
**Pull request by tkhmy**
## Description(required)
- fix ignore_emergency parameter not working

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: src/signage/src/signage/route_handler.py [hunk 1]
```diff
@@ -73,9 +73,10 @@ def __init__(
         self._node.create_timer(0.2, self.announce_engage_when_starting)
 
     def emergency_checker_callback(self):
-        if self._parameter.ignore_emergency:
-            in_emergency = False
-        if self._autoware.information.operation_mode == OperationModeState.STOP:
+        if (
+            self._parameter.ignore_emergency
```
Comment (yabuta) [inline src/signage/src/signage/route_handler.py:77]: ignore_emergencyがtrueの時_in_slowing_stateもFalseになるようにしてもらえますか？

**Review by yabuta — CHANGES_REQUESTED** [review]

File: src/signage/src/signage/route_handler.py (hunk 1 above)
Comment (tkhmy) [reply src/signage/src/signage/route_handler.py:77]: comfortable stop も無視したいですか？

Comment (yabuta) [reply src/signage/src/signage/route_handler.py:77]: そうですね。comfortable stopとpull overのときも無視したいと思います

Comment (tkhmy) [reply src/signage/src/signage/route_handler.py:77]: [fa95b37](https://github.com/tier4/AutowareSignage/pull/110/commits/fa95b3793ecb9a25d90d7901e16e562c8f0d63ec) で修正しました。

**Review by yabuta — APPROVED** [review]
LGTM

**Pull request merged**